crates/red_knot_test/README.md

```diff
@@ -8,8 +8,8 @@ under a certain directory as test suites.
 
 A Markdown test suite can contain any number of tests. A test consists of one or more embedded
 "files", each defined by a triple-backticks fenced code block. The code block must have a tag string
-specifying its language; currently only `py` (Python files) and `pyi` (type stub files) are
-supported.
+specifying its language. We currently support `py` (Python files) and `pyi` (type stub files), as
+well as typeshed `VERSIONS` files and `toml` for configuration.
```

Comment:
If you like, you could link to https://github.com/python/typeshed/blob/c546278aae47de0b2b664973da4edb613400f6ce/stdlib/VERSIONS#L1-L18 here, which explains the format of the `VERSIONS` file. And our parser for the `VERSIONS` file is at https://github.com/astral-sh/ruff/blob/main/crates/red_knot_python_semantic/src/module_resolver/typeshed.rs